org.abapgit.adt.ui/src/org/abapgit/adt/ui/internal/i18n/messages.properties

```diff
@@ -44,6 +44,8 @@ AbapGitWizard_task_cloning_repository=Linking repository...
 AbapGitWizard_task_pulling_repository=Pulling from repository...
 AbapGitWizard_title=Link abapGit Repository
 AbapGitWizardPull_title=Pull from abapGit Repository
+AbapGitWizardPullSelectedObjects_OverwriteObjectsMessage=The following objects have been modified (or deleted) locally. Select the objects which should be overwritten (or recreated).
```

Comment:
Description messages for pages handling the selection of modified objects to pull